src/Controller/HelperController.php

```diff
@@ -285,32 +285,26 @@ public function setObjectFieldValueAction(Request $request)
 
         // Handle entity choice association type, transforming the value into entity
         if ('' !== $value && 'choice' == $fieldDescription->getType() && $fieldDescription->getOption('class')) {
```

Comment:
`This yoda contitions I like not.` Could you maybe switch it? (if we do not have documented somewhere that we should not use yoda conditions just ignore.)

Reply:
If we use Symfony rules we have it: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/.php_cs.dist
Unless it was disabled manually, which is not.

Personally I do not like yoda conditions, I disable them manually on my project cs-fixer config, but if I flip this we will start getting errors on flintCI and travis

Reply:
Same, just ignore it then.